dist/debian/control.template

```diff
@@ -0,0 +1,16 @@
+Source: %{product}-cqlsh
+Section: misc
+Priority: extra
+Maintainer: Takuya ASADA <syuu@scylladb.com>
+Homepage: http://www.scylladb.com
+Standards-Version: 3.9.5
+Rules-Requires-Root: no
+
+Package: %{product}-cqlsh
+Architecture: all
+Depends: python3
+Recommends: ntp | time-daemon
+Conflicts: cassandra
+Description: Scylla cqlsh tool
+ Scylla is a highly scalable, eventually consistent, distributed,
+ partitioned row DB.
```

Comment:
Please adjust to explain that this is a shell for ScyllaDB.

Reply:
is this good enough:
```
 scylla-cqlsh is tool to connect to Scylla or Cassandra clusters, for debugging 
 and dynamically do queries from a CLI 
```

Reply:
suggest s/do/perform/